recipe/meta.yaml

```diff
@@ -30,14 +28,14 @@ requirements:
     - datasets !=2.5.0
```

Comment:
I don't see datasets and importlib-metadata set in https://github.com/huggingface/transformers/blob/main/setup.py#L423